.github/workflows/deploy.yml

```
@@ -2,7 +2,6 @@ name: Deploy
 on:
   release:
     types: [published]
-  workflow_dispatch:
```

```suggestion
  workflow_dispatch
```

We should leave this IMHO. For times we want to run a manual deployment.

---

I think it should always be run from the `Release` Action though, so I moved it from `deploy` to `release` Action - that way a release is not created on every commit/push but manually or on the cron (the changelogs will be much more interesting too)